website/playground/index.jsx

```diff
@@ -11,6 +11,41 @@ const {
   ReactDOM: { createRoot },
 } = window;
 
+function getInitialTheme() {
+  const saved = localStorage.getItem("theme");
+  if (saved) return saved;
+
+  return window.matchMedia("(prefers-color-scheme: dark)").matches
+    ? "dark"
+    : "light";
+}
+
+function applyTheme(theme) {
+  document.documentElement.setAttribute("data-theme", theme);
+}
+
+function ThemeToggle() {
+  const [theme, setTheme] = React.useState(getInitialTheme);
+
+  const toggleTheme = () => {
+    const newTheme = theme === "dark" ? "light" : "dark";
+    setTheme(newTheme);
+    localStorage.setItem("theme", newTheme);
+    applyTheme(newTheme);
+  };
+  applyTheme(theme);
+
+  return React.createElement(
+    "button",
+    {
+      onClick: toggleTheme,
+      className: "btn",
+      "aria-label": `Switch to ${theme === "dark" ? "light" : "dark"} theme`,
+    },
+    theme === "dark" ? "☀️" : "🌙",
```

Comment:
Can we use the same button as the home page?

<img width="756" height="94" alt="image" src="https://github.com/user-attachments/assets/c2403f3b-399c-4f32-927d-0f3f4ac27955" />
<img width="745" height="64" alt="image" src="https://github.com/user-attachments/assets/df23f8e3-7ab5-4845-997b-e71ad6ea74dc" />